modules/react/layout/lib/Grid.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {createComponent, StyledType} from '@workday/canvas-kit-react/common';
+import {Box, BoxProps} from './Box';
+
+import {grid, GridStyleProps} from './utils/grid';
+import {gridItem, GridItemStyleProps} from './utils/gridItem';
+
+export type GridProps = BoxProps & GridStyleProps;
+export type GridItemProps = BoxProps & GridItemStyleProps;
+
+const StyledGrid = styled(Box)<StyledType & GridProps>(
+  {
+    display: 'grid',
+  },
+  grid
+);
+
+const StyledGridItem = styled(Box)<StyledType & GridItemProps>(
+  {
+    display: 'grid',
```

Comment:
Does the child element also need `display: grid` ? or is it just the parent?

Reply:
hmm, I still don't think that `Grid.Item` needs `display: grid` doesn't that define another `grid` container? I'm looking at this for reference: https://css-tricks.com/snippets/css/complete-guide-grid/#aa-properties-for-the-childrengrid-items